scripts/output/single/ar6Climate.R

```diff
@@ -1,152 +1,234 @@
+# |  Contact: remind@pik-potsdam.de
+
+#' @title AR6 climate assessment
+#' @description Assessment of new emissions pathways consistent with the climate variable data from the working group
+#' III contribution to the IPCC Sixth Assessment (AR6) report. The script uses REMIND emissions data to perform
+#' harmonization and infilling based on the IIASA [climate-assessment](https://climate-assessment.readthedocs.io)
+#' package. The harmonized and infilled emissions are then used as input to the
+#' [MAGICC7 simple climate model](https://magicc.org/). Results are postprocessed and appended to the REMIND model
+#' intercomparison file (MIF).
+#'
+#' @author Gabriel Abrahao, Oliver Richters, Tonn Rüter
+#'
+#' @param outputdir Directory where REMIND MIF file is located. Output files generated in the process will be written
+#' to a subfolder "climate-assessment-data" in this directory. Defaults to "."
+#' @param gdxName Name of the REMIND GDX file. Defaults to "fulldata.gdx"
+
 library(madrat)
 library(remind2)
 library(quitte)
 library(piamInterfaces)
-library(yaml)
 library(lucode2)
-library(readxl) # GA: Wont be necessary after https://github.com/iiasa/climate-assessment/pull/43 goes into release
+library(yaml)
+library(tidyverse)
+library(readr)
+library(stringr)
 
 ############################# BASIC CONFIGURATION #############################
-gdx_name     <- "fulldata.gdx"             # name of the gdx
-cfg_name     <- "cfg.txt"                  # cfg file for getting file paths
 
+gdxName <- "fulldata.gdx"             # name of the gdx
+cfgName <- "cfg.txt"                  # cfg file for getting file paths
 
-if(!exists("source_include")) {
-   # Define arguments that can be read from command line
-   outputdir <- "."
-   readArgs("outputdir", "gdx_name", "gdx_ref_name", "gdx_refpolicycost_name")
+if (!exists("source_include")) {
+  # Define arguments that can be read from command line
+  outputdir <- "."
+  readArgs("outputdir", "gdxName", "gdx_ref_name", "gdx_refpolicycost_name")
 }
 
-gdx           <- file.path(outputdir, gdx_name)
-cfg_path      <- file.path(outputdir, cfg_name)
-logfile       <- file.path(outputdir, "climate.log") # specific log for python steps
-scenario <- getScenNames(outputdir)
-remind_reporting_file <- file.path(outputdir,paste0("REMIND_generic_", scenario,".mif"))
+cfgPath               <- file.path(outputdir, cfgName)
+logFile               <- file.path(outputdir, "log_climate.txt") # specific log for python steps
+scenario              <- getScenNames(outputdir)
+remindReportingFile   <- file.path(outputdir, paste0("REMIND_generic_", scenario, ".mif"))
+climateAssessmentEmi  <- normalizePath(file.path(outputdir, paste0("ar6_climate_assessment_", scenario, ".csv")),
+                                       mustWork = FALSE)
+# TODO: REMOVE THE NEXT LINE. piamInterfaces should be installed as a package on the cluster
+devtools::load_all("/p/tmp/tonnru/piamInterfaces/")
+devtools::load_all("/p/tmp/tonnru/quitte/")
```

Comment:
The new version should normally be installed on the cluster within minutes after you merge your PR on github.
```suggestion
library(piamInterfaces)
library(quitte)
```